src/main/java/Duke/Parser.java

```diff
@@ -0,0 +1,75 @@
+package Duke;
+
+public class Parser {
+
+    public enum Mode{
+        LIST, BYE, DONE, DELETE, TODO, DEADLINE, EVENT;
+
+    }
+
+    public Mode mode(String input) throws DukeException {
+        String command = input.split(" ")[0];
+        switch (command) {
+            case "todo":
+                return Mode.TODO;
+            case "deadline":
+                return Mode.DEADLINE;
+            case "event":
+                return Mode.EVENT;
+            case "bye":
+                return Mode.BYE;
+            case "done":
+                return Mode.DONE;
+            case "delete":
+                return Mode.DELETE;
+            case "list":
+                return Mode.LIST;
+            default:
+                throw new DukeException("\u2639 OOPS!!! I'm sorry, but I don't know what that means :-(");

```

Comment:
Case statements should not have extra indentation. (they should be placed on the same indentation as the switch statement)